configuration/debian/tedge/postrm

```diff
@@ -1,8 +1,36 @@
 #!/bin/sh
 set -e
 
+
+### Remove user tedge
+if getent passwd tedge >/dev/null; then
+    deluser --quiet --system tedge
+fi
+
 ### Remove configs
+if [ -d "/etc/tedge" ]; then
+    rm -rf /etc/tedge
```

Comment:
The config files must be removed only on purge.

Notably, this directory contains:
* the `tegde.toml` that has been tuned for the device.
* device certificates that might have been acquired from a tier.